localization/english/culture/traditions/wc_cultural_traditions_l_english.yml

```
@@ -0,0 +1,19 @@
+﻿l_english:
+ # Traditions
+ tradition_arkonite_name:0 "Arkonite"
+ tradition_arkonite_desc:0 "This culture uses Arkonite daily in their society. From powering arcane devices and light sources, to helping them channel magics."
```

I'd get rid of the "this culture uses" phrase.
```suggestion
 tradition_arkonite_desc:0 "Arkonite has a multitude of uses, be it in everyday life, or on more special occasions. When handled correctly, it is a great channeling material for light sources and arcane devices."
```